metadata-ingestion/src/datahub/ingestion/source/redshift/query.py

```diff
@@ -10,21 +10,19 @@ class RedshiftQuery:
         AND (datname <> ('template1')::name)
         """
 
-    list_schemas: str = """SELECT database_name,
-        schema_name,
-        schema_type,
-        -- setting user_name to null as we don't use it now now and it breaks backward compatibility due to additional permission need
-        -- usename as schema_owner_name,
+    list_schemas: str = """SELECT distinct n.nspname AS "schema_name",
+        'local' as schema_type,
```

Comment:
What's this?

Reply:
This is what the https://docs.aws.amazon.com/redshift/latest/dg/r_SVV_REDSHIFT_SCHEMAS.html returns normally but don't use it yet.